Update ADK doc according to issue #1350 - 9 (#1367)
google/adk-docs: docs/agents/models/litellm.md

translation layer for models and model hosting services, providing a
standardized, OpenAI-compatible interface to over 100+ LLMs. ADK provides
integration through the LiteLLM library, allowing you to access a vast range of

> LLMs from providers like Anthropic, Open AI, xAI, VertexAI, NVIDIA, OLlama, DeepSeek, and many

Reviewer: This update was triggered by https://github.com/google/adk-python/commit/d5332f44347f44d60360e14205a2342a0c990d66, which expanded ADK's LiteLLM's auto-routing prefixes.

Let's use "OpenAI, Anthropic, Ollama, Mistral, DeepSeek, and Cohere", which are all in ADK's LiteLLM support list.

Also FYI, we should never reference "Vertex AI" anymore  since it was renamed to Agent Platform, but that's moot once it's removed from this list.

```suggestion
LLMs from providers such as OpenAI, Anthropic, Ollama, Mistral, DeepSeek, and Cohere, and many
```

Reviewer 2: making a note about Vertex AI, thanks for your review
